README.md

```diff
@@ -1 +1,233 @@
-# Online-web-development-course
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
Nice README 📖 but make sure to remove all the comments to make the file look more professional, the comments are here to guide you and once you're done with the README it's not useful anymore. 🙅🏽

Reply:
Thanks for your guidance and suggestion even for the ideas , I fix that one also